src/main/kotlin/org/eclipse/lmos/operator/util/SubsetProvider.kt

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.lmos.operator.util  
+
+object SubsetProvider {  
+
+    private val canaryReleaseEnabled: Boolean by lazy {  
+        val envValue = System.getenv("LMOS_OPERATOR_CANARY_RELEASE_ENABLED")  
```

Comment:
As the lmos-operator is built using spring boot, I asked myself whether we could (and should) use spring boot's configuration to inject the "canaryReleaseEnabled" flag (instead of bypassing spring boot and reading from system environment directly).

I see that it is currently not immediately possible, because the `SubsetProvider` is used by the `ChannelDependentResource` (via the `RoutingChannelGenerator`), and the `ChannelDependentResource` is currently not managed by spring. But if we marked it as a `@Component`, we could inject the flag via spring boot?